object-cache.php

```diff
@@ -527,6 +533,58 @@ function get_multi( $groups ) {
 		return $return;
 	}
 
+	public function get_multiple( $ids, $group = 'default', $force = false ) {
+		$mc = $this->get_mc( $group );
+
+		$no_mc = in_array( $group, $this->no_mc_groups );
+
+		$uncached_keys = array();
+
+		$return = array();
+		$return_cache = array();
+
+		foreach ( $ids as $id ) {
+			$key = $this->key( $id, $group );
+
+			if ( isset( $this->cache[ $key ] ) && ( ! $force || $no_mc ) ) {
+				$value         = $this->cache[ $key ]['found'] ? $this->cache[ $key ]['value'] : false;
+				$return[ $id ] = is_object( $value ) ? clone $value : $value;
+			} else if ( $no_mc ) {
+				$return[ $id ]        = false;
+				$return_cache[ $key ] = [
+					'value' => false,
+					'found' => false,
+				];
+			} else {
+				$uncached_keys[ $id ] = $key;
+			}
+		}
+
+		if ( $uncached_keys ) {
+			$this->timer_start();
+			$uncached_keys_list = array_values( $uncached_keys );
+			$values             = $mc->get( $uncached_keys_list );
+			$elapsed            = $this->timer_stop();
+
+			$this->group_ops_stats( 'get_multiple', $uncached_keys_list, $group, null, $elapsed );
+
+			foreach ( $uncached_keys as $id => $key ) {
+				$found = array_key_exists( $key, $values );
+				$value = $found ? $values[ $key ] : false;
+
+				$return[ $id ]        = $value;
+				$return_cache[ $key ] = [
+					'value' => $value,
+					'found' => $found,
```

Comment:
`false !== $value` will produce an incorrect result if we have stored `false` for the given key.